Fix #3154 - Fixes the UI inconsistency in README.md file

OHassini opened the pull request:
## Description
Fixes the layout of the first and the last step in the "Building from source" section in the README.md
file.
## Related issues and discussion
#3154 

## Screenshots, if any
A summary of the changes :
![Capture d’écran (167)](https://user-images.githubusercontent.com/38397893/70231579-e5e97800-175a-11ea-9e27-dfec2d3bb7c2.png)

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Validate the JS and HTML files with `grunt validate` to detect errors and potential problems in JavaScript code.

- [x] Run the tests by opening `test/SpecRunner.html` in the browser to make sure you didn't break anything.

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `community-app/Contributing.md`.

OHassini committed:
Fix #3154 - Fixes the UI inconsistency in README.md file

fix

fixe

fix

OHassini commented:
@luckyman20 Hey : ), Please have a look at this.

luckyman20 commented on README.md:
````diff
@@ -54,8 +58,14 @@ This is the default web application built on top of the MifosX platform for the
 
-1. Default username/password: mifos/password. This application will hit the demo server by default.
+   ```
+    Username: fineractCn
````
```suggestion
    Username: mifos
```

OHassini commented:
Oh my god, how fool of me, I am really sorry I was working on a mifos-mobile-cn task right before making this commit, I just updated the README file again with the changes and squashed my commits. @luckyman20

luckyman20 commented on README.md:
````diff
@@ -54,8 +58,14 @@ This is the default web application built on top of the MifosX platform for the
+    Username: mifos
+    Password: password
+   ```
+This application will hit the demo server by default.
````
Keep this line at the end as a note.

OussEmaDevCode replied:
the end of the README file or the "Building from source" section ?

OHassini commented:
@luckyman20 done!!

luckyman20 reviewed (approved):
Looks good!
